test/extensions/filters/common/expr/context_test.cc

```diff
@@ -30,6 +30,13 @@ TEST(Context, EmptyHeadersAttributes) {
   EXPECT_TRUE(headers.empty());
 }
 
+TEST(Context, InvalidRequest) {
+  Http::TestRequestHeaderMapImpl header_map{{"referer", "dogs.com"}};
+  HeadersWrapper<Http::RequestHeaderMap> headers(&header_map);
+  auto header = headers[CelValue::CreateStringView("dogs.com/n")];
```

Comment:
oops, i think this is a mistake `/n` is ok, `\n` is newline.

Reply:
Yup, you're right - test still passes, which is good!